py/desispec/parallel.py

```diff
@@ -147,41 +157,58 @@ def dist_discrete(worksizes, workers, id, pow=1.0):
     return dist[id]
 
 
-
 @contextmanager
-def stdouterr_redirected(to=os.devnull, comm=None):
+def stdouterr_redirected(to=None, comm=None):
```

Comment:
Please update docstring to clarify whether `to` is a file name, open file handle, either, etc., what happens in the default case, and also what `comm` is (I know, but future developers may not...)